config/ons.yml

```diff
@@ -16,7 +17,10 @@ entries:
   replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/Older_deprecated/ons_release_v1.0.owl
 
 - exact: /ons_release_v1.1.owl
-  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/ons_release_v1.1.owl
+  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/Older_deprecated/ons_release_v1.1.owl
+
+- exact: /ons_release_v1.0.obo
+  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/Older_deprecated/ons_release_v1.1.obo
   
 - prefix: /imports/
-  replacement: https://github.com/enpadasi/Ontology-for-Nutritional-Studies/tree/master/src/ontology/imports
+  replacement: https://raw.githubusercontent.com/enpadasi/Ontology-for-Nutritional-Studies/master/src/ontology/imports/ONS_imports.owl
```

Comment:
This is probably not working. I dont think you need to make this change at all, or at least no include `ONS_imports.owl` here. In your ontology (ONS) you actually hard code the raw github location in the import (which you probably should not), but I would remove `ONS_imports.owl` and the purl to http://purl.obolibrary.org/obo/ons/imports/OMS_imports.owl will work just fine.

Reply:
Ok, I'll revert back to how it was

- prefix: /imports/
  replacement: https://github.com/enpadasi/Ontology-for-Nutritional-Studies/tree/master/src/ontology/imports

or you would suggest removing it entirely?

Reply:
I updated it now the way I think it should be! What do you think?

Reply:
Ok the change is fine, thank you for the help. Getting the hang of a lot of things here, github wonders included.